curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ee6e3a242da6bd579de69.md

```diff
@@ -19,7 +19,7 @@ let hello;
 
 Variable naming follows specific rules: names can include letters, numbers, dollar signs, and underscores, but cannot contain spaces and must not begin with a number.
 
-Declare a `character` variable in your code. 
+Use the `let` keyword to declare a variable called `character`.
```

Comment:
please put it back to its original form, keep solution consistent to the ticket description and solution provided

Reply:
Both the issues I worked on are in this PR (Issue https://github.com/freeCodeCamp/freeCodeCamp/issues/54930 & https://github.com/freeCodeCamp/freeCodeCamp/issues/54926).  This file is part of issue #54926. Per the ticket descriptions and solution provided it matches the issue.  When I tried to submit a second PR for this second issue,  it had already been committed to this PR.  Did I complete the PR incorrectly? 

Should I go ahead and revert back to the original for issue #54926?

Reply:
okay in this case, keep it as it is then, my bad, didn't see both issues linked together.

Reply:
> Both the issues I worked on are in this PR (Issue #54930 & #54926). This file is part of issue #54926. Per the ticket descriptions and solution provided it matches the issue. When I tried to submit a second PR for this second issue, it had already been committed to this PR. Did I complete the PR incorrectly?
> 
> Should I go ahead and revert back to the original for issue #54926?

That's caused by making both changes on the same branch - `main`. It's recommended to create new, separate branch for each separate change (issue), while keeping the `main` branch in sync with the `freeCodeCamp:main` branch. That way it's always possible to create new _clean_ branch off the `main`, so changes from different branches don't mix up.

For similar reason usually it's better to keep single PR for single issue (unless single change completes multiple issues). As there was in the meantime submitted another PR, making changes for #54930, it'd be great if you could revert change for #54930, and leave one for #54926. Making your PR close #54926, while the other PR closes #54930.